ext/dev.js

```diff
@@ -117,10 +117,14 @@
 			}
 		},
 		_logger:function(type, arr){
-			if(console.log.apply){
-				console[type].apply(console, arr);
-			} else {
+			if (typeof console.log == "object") {
```

Comment:
This is feature detection, specifically for IE8. console.log is an object in IE8 but a function in all other modern browsers. console.log.apply does not exist in IE8